src/newrelic_telemetry_sdk/client.py

```diff
@@ -161,7 +161,6 @@ def __init__(self, license_key, host=None, port=443):
             port=port,
             retries=retries,
             headers=headers,
-            strict=True,
```

Comment:
Documenting here that this param was Python 2 only, with the removal of Python 2 there's no need to add it anymore.